pages/project/[id]/edit/contributors.tsx

```diff
@@ -0,0 +1,82 @@
+import { NextPageWithLayout } from "pages/_app";
+
+// Form
+import { yupResolver } from "@hookform/resolvers/yup";
+import ContributorsStep, {
+  contributorsStepSchema,
+  ContributorsStepValues,
+} from "components/partials/create/project/steps/ContributorsStep";
+import { useForm } from "react-hook-form";
+import * as yup from "yup";
+
+// Components
+import EditProjectLayout from "components/layout/EditProjectLayout";
+import FetchProjectLayout, { useProject } from "components/layout/FetchProjectLayout";
+import Layout from "components/layout/Layout";
+import EditFormLayout from "components/partials/project/edit/EditFormLayout";
+
+//
+
+export interface EditContributorsValues {
+  contributors: ContributorsStepValues;
+}
+
+const EditContributors: NextPageWithLayout = () => {
+  const project = useProject();
+
+  /* Form setup */
+
+  const defaultValues: EditContributorsValues = {
+    contributors: project.metadata.contributors || [],
```

Comment:
Seems that an empty array/list is not the same as undefined on the Zenflows backend, can you double check this?

Reply:
Yes, it's true. I'll fix this

Reply:
It works also with empty arrays now :)